guides/v2.3/graphql/reference/paypal-create-payflow-pro-token.md

```diff
@@ -0,0 +1,67 @@
+---
+group: graphql
+title: getPayflowLinkToken query
+---
+
+The `getPayflowLinkToken` query retrieves PayPal payment credentials for a PayPal Payflow transaction. You must run this query after you [set the payment method]({{ page.baseurl}}/graphql/reference/quote-payment-method.html) and [place the order]({{ page.baseurl}}/graphql/reference/quote-place-order.html).
+
+See [Paypal Payflow Link payment method]({{page.baseurl}}/graphql/payment-methods/payflow-link.html) for detailed information about the workflow of PayPal Payflow transactions.
+
+## Syntax
+
+`getPayflowLinkToken(input: PayflowLinkTokenInput): PayflowLinkToken`
+
+## Example
+
+The following example requests a token in a Payflow Link transaction.
+
+**Request**
+
+```text
+{
+  getPayflowLinkToken(input: {cart_id: "123"}) {
+    secure_token
+    secure_token_id
+    mode
+    paypal_url
+  }
+}
+```
+
+**Response**
+
+```json
+{
+    "data": {
+        "getPayflowLinkToken": {
+            "secure_token": "<token-value>",
+            "secure_token_id": "<token-value-id>",
+            "mode": "TEST",
+            "paypal_url": "https://pilot-payflowlink.paypal.com"
+        }
+    }
+}
+```
+
+## Input attributes
+
+### PayflowLinkTokenInput {#PayflowLinkTokenInput}
+
+The `PayflowLinkTokenInput` object defines the attributes required to receive a Payflow Link  token from PayPal.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`cart_id` | String! | The unique ID that identifies the customer's cart
+
+## Output attributes
+
+### PayflowLinkToken
```

Comment:
```suggestion
### PaypalExpressToken
```
?

Reply:
Actually, the heading is correct, and the text is wrong.